wordfreq/tokens.py

```diff
@@ -115,8 +117,17 @@ def tokenize(text, lang):
     if lang == 'ja':
         global mecab_tokenize
         if mecab_tokenize is None:
-            from wordfreq.mecab import mecab_tokenize
-        return mecab_tokenize(text)
+            from wordfreq.japanese import mecab_tokenize
+        tokens = mecab_tokenize(text)
+        return [token.casefold() for token in tokens if TOKEN_RE.match(token)]
+
+    if lang == 'zh':
+        global jieba_tokenize
+        if jieba_tokenize is None:
+            from wordfreq.chinese import jieba_tokenize
+        tokens = jieba_tokenize(text)
+        return [token.casefold() for token in tokens if TOKEN_RE.match(token)]
```

Comment:
This block has me wondering if we should unify the Japanese and Chinese files into something that will handle the delayed imports (etc.) internally.